include/crypto/aes_platform.h

```diff
@@ -81,9 +81,13 @@ size_t ppc_aes_gcm_encrypt(const unsigned char *in, unsigned char *out, size_t l
                          const void *key, unsigned char ivec[16], u64 *Xi);
 size_t ppc_aes_gcm_decrypt(const unsigned char *in, unsigned char *out, size_t len,
                          const void *key, unsigned char ivec[16], u64 *Xi);
+size_t ppc_aes_gcm_encrypt_wrap(const unsigned char *in, unsigned char *out, size_t len,
+                         const void *key, unsigned char ivec[16], u64 *Xi);
```

Comment:
indentation is wrong here.. the const needs to line up with the one above..

Reply:
Yeah, but the indentation on previous lines is wrong too. Should those be fixed?